src/functionBased/components/TodoItem.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable */
+import React, { useState, useEffect } from 'react';
```

Comment:
* Avoid disabling eslint as this will help you learn how to fix find and fix errors and improve on your code quality